package.json

```diff
@@ -24,7 +24,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mozilla/firefox-launch.git"
+    "url": "git+https://github.com/mozilla-extensions/firefox-bridge"
   },
   "keywords": [
```

Comment:
You can probably omit the keywords.

Reply:
Since I cannot comment on all lines, is the `directories` field required? If not, please remove it.

Same question, for `main`.

Reply:
The `description` should also be fixed:

```
Extension that enables the user to launch Firefox from another browser
```